apps/spruce/src/pages/waterfall/WaterfallGrid.tsx

```diff
@@ -188,6 +232,7 @@ export const WaterfallGrid: React.FC<WaterfallGridProps> = ({
               </InactiveVersion>
             );
           })}
+          {isPending && <FetchMoreLoader />}
```

Comment:
There might be some sort of race happening where the FetchMoreLoader shows up even if we have a full page of tasks loaded.
<img width="1396" alt="image" src="https://github.com/user-attachments/assets/c0f18acf-f973-4ef9-ac6e-1f94530ca5b1" />